pupil_src/shared_modules/pupil_producers.py

```diff
@@ -301,23 +302,23 @@ def start_eye_process(self, eye_id):
             for ext in (".mjpeg", ".mp4", ".mkv")
         ]
         existing_locs = [loc for loc in potential_locs if os.path.exists(loc)]
-        timestamps_path = os.path.join(
-            self.g_pool.rec_dir, "eye{}_timestamps.npy".format(eye_id)
-        )
+        timestamps_path_lst = glob.glob(os.path.join(
+            self.g_pool.rec_dir, "eye{}*_timestamps.npy".format(eye_id)))
+        timestamp_len = sum(len(np.load(t)) for t in timestamps_path_lst)
```

Comment:
We should load the lookup table and filter it as if `fill_gaps==False`. Loading the single timestamp files does not let you know if there are broken containers.